core/templates/dev/head/pages/exploration-editor-page/improvements-tab/answer-details-improvement-card/answer-details-improvement-card.directive.html

```diff
@@ -0,0 +1,9 @@
+<div>
+  <b>State Name:</b>
+  <span><[getStateName()]></span>
+</div>
+<br>
+<div>
+  <b>No. of answer details recieved:</b>
```

Comment:
- Typo: s/recieved/received
- Avoid abbreviations, reword as: `Number of answer details received:`

Reply:
Done